datasets/bbaw_egyptian/README.md

```diff
@@ -0,0 +1,169 @@
+---
+annotations_creators:
+- specialized egyptologists
+language_creators:
+- found
+languages:
+- de, en, eg
+licenses:
+- cc-by-4.0
+multilinguality:
+- multilingual
+size_categories:
+- 100K<n<1000K
+source_datasets:
+- extended|wikipedia
+task_categories:
+- translation
```

Comment:
`specialized egyptologists` is not a valid `annotations_creators` tag. You can use this instead:
```
annotations_creators:
- expert-generated
```
There is a tool to create those tags [here](https://huggingface.co/datasets/tagging/).

For the languages, there should be one language per line:
```
- de
- en
- eg
```

Finally the `task_ids` tags are missing:
```
task_categories:
- conditional-text-generation
task_ids:
- machine-translation
```